tests/venv/py35/README.rst

```diff
@@ -0,0 +1,8 @@
+System - Specifying runtime environments
```

Comment:
Note that these READMEs are mostly just for building the docs...if a test doesn't have a README file then it just won't be added to the docs, so for tests that duplicate functionality (but are there for test coverage purposes) I'm fine not including READMEs in them

Reply:
I didn't realize that's what these readmes were for! I'll look at these again to see if they should be changed or just removed.

Reply:
Also note #300 which updates a bunch of the READMEs